src/hooks.server.ts

```diff
@@ -7,7 +7,6 @@ const headers: Handle = async function handle({ event, resolve }) {
 	response.headers.set('Strict-Transport-Security', 'max-age=31536000; includeSubDomains; preload');
 	response.headers.set('X-Content-Type-Options', 'nosniff');
 	response.headers.set('X-Frame-Options', 'DENY');
-	response.headers.set('Permissions-Policy', '*');
```

Comment:
this was causing a warning in the console, and looking into it its not valid to have the value be `*`